docs/Configuration.md

```diff
@@ -52,6 +52,11 @@ builder:
       afterTask: custom-task-name-1
       configuration:
         color: blue
+  resources:
+    excludes:
+      - /resources/some/project/name/test_results/**
+      - /test-resources/**
+      - !/test-resources/some/project/name/demo-app/**
```

Comment:
According to the syntax highlighting, this is not a valid string.
When parsing it, the JSON content is `null`. Quotes should be used.

Reply:
Done 👍